### PR TITLE
Add Ukrainian translation for Python keywords and built-in functions

### PR DESCRIPTION
**Pull request by Jam-Iko**

### CODE REVIEW

**Comment by madiedgar** [comment]
Congrats on merging your pull request @Jam-Iko! 🎉🎉🎉 ❤️ THANK YOU THANK YOU THANK YOU ❤️ We'd love to celebrate you and your contribution to the community. Let us say "Thank You" again, by sending you a [special gift](https://madi357705.typeform.com/to/wzub4R)!

We couldn't thank you enough for your contribution. We hope you continue sharing your gifts and knowledge with not just us, but with the world around you.😍

File: locale/uk.yml [hunk 1]
```diff
@@ -1247,114 +1247,128 @@ __TRAIT__: ""
...
+  async: "асинхронно" # can also be "корутинно"
+  await: "очікувати"
+  break: "перервати"
+  class: "клас"
+  continue: "продовжити"
+  def: "оголосити"
+  del: "видалити" # can also be "видал"
+  elif: "інакше_якщо"
+  else: "інакше"
+  except: "виняток" # can also be "окрім"
```
[the remark this answers is not in the document]
Comment (Jam-Iko) [reply locale/uk.yml]: Perhaps, something along the lines of “перехопити”?

File: locale/uk.yml [hunk 2]
```diff
@@ -1247,114 +1247,128 @@ __TRAIT__: ""
...
+  is: "є"
+  lambda: "лямбда"
+  nonlocal: "нелокальна"
+  not: "не"
+  or: "або"
+  pass: "пропустити"
+  raise: "підняти"
+  return: "повернути"
+  try: "спробувати"
+  while: "допоки"
```
[the remark this answers is not in the document]
Comment (Jam-Iko) [reply locale/uk.yml]: Okay.

File: locale/uk.yml [hunk 3]
```diff
@@ -1247,114 +1247,128 @@ __TRAIT__: ""
...
+  with: "разом_з"
```
[the remark this answers is not in the document]
Comment (Jam-Iko) [reply locale/uk.yml]: Sounds better.

File: locale/uk.yml [hunk 4]
```diff
@@ -1247,114 +1247,128 @@ __TRAIT__: ""
...
-  tuple: ""
-  type: ""
-  vars: ""
-  zip: ""
-  __import__: ""
+  abs: "абсолютне_значення" # can also be "абс"
+  all: "всі"
+  any: "будь_які"
+  ascii: "ascii"
+  bin: "двійкове" # can also be "двійкове_значення", "бінарне"
```
[the remark this answers is not in the document]
Comment (Jam-Iko) [reply locale/uk.yml]: Okay.

File: locale/uk.yml [hunk 5]
```diff
@@ -1247,114 +1247,128 @@ __TRAIT__: ""
...
+  chr: "символ"
+  classmethod: "метод_класу"
+  compile: "компілювати"
+  complex: "комплексне"
+  delattr: "видалити_атрибут" # can also be "видалити_поле"
+  dict: "словник"
+  dir: "директорія" # can also be "дир"
+  divmod: "ділення_націло" # can also be "діл_націло"
+  enumerate: "перерахувати"
+  eval: "оцінити"
```
[the remark this answers is not in the document]
Comment (Jam-Iko) [reply locale/uk.yml]: I guess I would go with «обчислити».

File: locale/uk.yml [hunk 6]
```diff
@@ -1247,114 +1247,128 @@ __TRAIT__: ""
...
+  exec: "виконати"
+  filter: "відфільтрувати"
```
[the remark this answers is not in the document]
Comment (Jam-Iko) [reply locale/uk.yml]: Okay.

File: locale/uk.yml [hunk 7]
```diff
@@ -1247,114 +1247,128 @@ __TRAIT__: ""
...
+  getattr: "отримати_атрибут" # can also be "отримати_поле"
+  globals: "глобальні"
+  hasattr: "має_атрибут" # can also be "має_поле"
+  hash: "закодувати"
+  help: "допомога"
+  hex: "шістнадцяткове" # can also be "шістнадцяткове_значення" 
+  id: "ід" # can also be "ідентифікатор"
+  input: "введення"
+  int: "ціле_число"
+  isinstance: "екземпляр"
```
[the remark this answers is not in the document]
Comment (Jam-Iko) [reply locale/uk.yml]: Yes, I think it is better :)

File: locale/uk.yml [hunk 8]
```diff
@@ -1247,114 +1247,128 @@ __TRAIT__: ""
...
+  issubclass: "підклас"
```
[the remark this answers is not in the document]
Comment (Jam-Iko) [reply locale/uk.yml]: Yes!

**Comment by Jam-Iko** [comment]
@JekRock thank you for your comments! It was rather challenging for me to come up with keyword translations on my own, so it is good to have another person’s perspective.

File: locale/uk.yml [hunk 9]
```diff
@@ -1247,114 +1247,128 @@ __TRAIT__: ""
...
-  try: ""
-  while: ""
-  with: ""
-  yield: ""
+  False: "хибність" # can also be "хиба"
+  None: "нічого"
+  True: "істина"
+  and: "і"
+  as: "як"
+  assert: "підтвердити" # can also be "перевірити"
```
[the remark this answers is not in the document]
Comment (Jam-Iko) [reply locale/uk.yml]: Okay.

File: locale/uk.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (JekRock) [reply locale/uk.yml]: Yes, something like that. It's not the correct translation but it's much better than a translation that doesn't reflect the meaning.

**Comment by JekRock** [comment]
@Jam-Iko always glad to help!
The translation of keywords is quite difficult.
Let me know if there is anything else I can help with.

**Comment by Jam-Iko** [comment]
@JekRock, I am not sure about the logistics of introducing the changes we have discussed. Do you want to create a PR to include these suggestions? Or I could create a new PR and you can review the translation and mark it ✅ as per issue #36?

**Comment by JekRock** [comment]
@Jam-Iko I think you should create a PR if you don't mind, of course. And I can review it then

**Pull request merged**